pkg/apis/projectcalico/v3/packetcapture.go

```diff
@@ -58,6 +58,8 @@ type PacketCaptureSpec struct {
 	// 	! has(label_name)
 	Selector string `json:"selector,omitempty" validate:"selector"`
 
+	// +kubebuilder:validation:MaxItems:=65535
```

Comment:
If these aren't doing anything I'd be inclined to remove them.

Did you see if this produced anything on the api server side?

Reply:
I am waiting on the https://github.com/tigera/apiserver/pull/464 to be merged so I can checked that.

Reply:
@asincu fyi it's being moved to https://github.com/tigera/apiserver/pull/465